common/containers/Tabs/Contracts/index.jsx

```diff
@@ -60,32 +60,30 @@ class Contracts extends Component {
     }
 
     return (
-      <section className="container" style={{ minHeight: '50%' }}>
-        <div className="tab-content">
-          <main className="tab-pane active" role="main">
-            <div className="Contracts">
-              <h1 className="Contracts-header">
-                <button
-                  className={`Contracts-header-tab ${interactActive}`}
-                  onClick={this.changeTab.bind(this, 'interact')}
-                >
-                  {translate('NAV_InteractContract')}
-                </button>{' '}
-                <span>or</span>{' '}
-                <button
-                  className={`Contracts-header-tab ${deployActive}`}
-                  onClick={this.changeTab.bind(this, 'deploy')}
-                >
-                  {translate('NAV_DeployContract')}
-                </button>
-              </h1>
-
-              <div className="Contracts-content">
-                {content}
-              </div>
-            </div>
-          </main>
+      <section className="Tab-content Contracts">
+        <div className="Tab-content-pane">
+          <h1 className="Contracts-header">
+            <button
+              className={`Contracts-header-tab ${interactActive}`}
+              onClick={this.changeTab.bind(this, 'interact')}
```

Comment:
Instead of binding `changeTab` to `this` each time, you could either bind it in the contructor or make it a property bound to an arrow function via ES7 property initializers to avoid making a new function every time you need to call it

Reply:
I'm binding not for `this`, but to have `interact` and `deploy` be sent as the first argument. I could use arrow functions here, but it just transpiles down to bind anyway. I'll go ahead and change to arrow fns for succinctness though!